src/checkout-buttons/strategies/apple-pay/apple-pay-button-strategy.ts

```diff
@@ -16,6 +16,7 @@ import { RemoteCheckoutActionCreator } from '../../../remote-checkout';
 import { ConsignmentActionCreator, ShippingOption } from '../../../shipping';
 import { CheckoutButtonInitializeOptions } from '../../checkout-button-options';
 import CheckoutButtonStrategy from '../checkout-button-strategy';
+import ApplePayShippingMethod = ApplePayJS.ApplePayShippingMethod;
```

Comment:
Why not use `ApplePayJS.ApplePayShippingMethod` in line number 234, and same for customer-strategy?

Reply:
Hi @animesh1987, thanks for your review and advice, I will change the code according to your suggestion.